.env.example

```
@@ -32,6 +32,12 @@ OMNI_MAX_CONNECTIONS_PER_IP=10
 OMNI_MAX_REQUESTS_PER_SECOND_PER_IP=100
 OMNI_MAX_REQUESTS_PER_SECOND_PER_IDENTITY=200
 
+# Prometheus Metrics (optional - enabled by default)
+# Exposes metrics at http://localhost:9090/metrics for Prometheus scraping
+METRICS_ENABLED=true
+METRICS_PORT=9090
+METRICS_HOST=0.0.0.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The METRICS_HOST key should go before the METRICS_PORT key

(UnorderedKey)